guides/common/modules/con_understanding-multiple-content-view-environments.adoc

```diff
@@ -0,0 +1,24 @@
+[id="understanding_multiple_content_view_environments_{context}""]
+= Understanding multiple content view environments
```

Comment:
I recommend changing the title to "Managing multiple content views" because most of the modules are procedures, not concepts, and because the wording would be consistent with the previous section, "Managing content views."